scene/2d/area_2d.cpp

```diff
@@ -261,8 +261,9 @@ void Area2D::_area_inout(int p_status, const RID &p_area, int p_instance, int p_
 
 	Map<ObjectID, AreaState>::Element *E = area_map.find(objid);
 
-	ERR_FAIL_COND(!area_in && !E);
-
+	if (!body_in && !E) {
```

Comment:
Should be `area_in`. This doesn't build.

Reply:
My bad, fixed